lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -30,15 +31,19 @@ export class FetchClient implements INetworkModule {
      * @param headers 
      * @param body 
      */
-    async sendPostRequestAsync<T>(url: string, options?: NetworkRequestOptions): Promise<T> {
+    async sendPostRequestAsync<T>(url: string, options?: NetworkRequestOptions): Promise<NetworkResponse<T>> {
```

Comment:
Should we also be wrapping `sendGetRequestAsync` response in NetworkResponse?

Reply:
Yep